playbooks/globalVariables.json

```diff
@@ -14,8 +14,8 @@
     {
         "id": 11,
         "name": "Excludelist_Domains",
-        "value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com",
-        "default_value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com"
+        "value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com,fortinet.com,twitter.com,facebook.com,linkedin.com,instagram.com",
+        "default_value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com,fortinet.com,twitter.com,facebook.com,linkedin.com,instagram.com"
```

Comment:
add fortiguard.com , forticloud.com  as well

Reply:
@cs-pooja-singh @cs-tushar-kanade Added fortiguard.com , forticloud.com in globalVariable.json and Create Default Global Variable.json